vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,17 +115,45 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
+    private Map<String, List<String>> exactDataTable(CellRangeAddress address, Sheet sheet)
+    {
+        Map<String, List<String>> mapData = new HashMap<>();
```

Comment:
Shouldn't it be part of ExcelSheetParser?

Reply:
remove it